src/flamegraph/color.rs

```diff
@@ -34,9 +35,18 @@ pub enum Palette {
     Orange,
 }
 
+#[derive(Debug, PartialEq)]
```

Comment:
And also `derive(Copy, Clone)` on this.

Reply:
[Ok](https://github.com/jonhoo/inferno/pull/32/commits/40386d59b8986d0089d0c998c339915be353062a)